manifests/params.pp

```diff
@@ -10,7 +10,7 @@
   $package_ensure    = 'latest'
   $ui_package_name   = 'consul_ui'
   $ui_package_ensure = 'latest'
-  $version = '0.5.0'
+  $version = hiera('consul::version', '0.5.0')  # ensure defaults resolve from hiera as well
```

Comment:
> Oh, do you mean those urls as params in init.pp?
Not sure what you mean, but end-result is that if I only set consul::version in hiera, my download comes from the version-number as set in params. I'll test it again!